Slider/Assets/Scripts/Sliders/Grid/AreaGrids/CaveGrid.cs

```diff
@@ -113,10 +116,14 @@ public void FinishCaves()
     public override void Save() 
     {
         base.Save();
+        caveDoor.Save();
```

Comment:
if `CaveDoor` implements `ISavable`, then its `Save()` should automatically get called